src/datasource.js

```diff
@@ -111,4 +108,13 @@ export class GenericDatasource {
       });
     });
   }
+
+  resolveVariables(target) {
+    var result = this.templateSrv.replace(target, this.templateSrv.variables);
+    // result might be in like "{id1,id2,id3}" (as string)
```

Comment:
Yes, it's the purpose of a PR opened on grafana (but still not reviewed) https://github.com/grafana/grafana/pull/5903
I was not the only plugin developer to ask some more convenient way to retrieve template variables.